package.json

```diff
@@ -48,7 +48,7 @@
     "angular-ui-bootstrap": "^2.5.0",
     "angular-vs-repeat": "^1.1.7",
     "angular2react": "^3.0.2",
-    "antd": "^3.12.3",
+    "antd": "^3.19.7",
```

Comment:
Woohoo!

Reply:
Plz do a site-wide sweep to see that no ant component broke.

Reply:
Biggest change seems to be the Table sorting arrows that are now blue and aligned with the text ([Percy](https://percy.io/Redash/redash/builds/2222641/view/144507643/1280?mode=diff&browser=chrome&snapshot=144507643)):
![image](https://user-images.githubusercontent.com/3356951/61556691-c61a1780-aa38-11e9-94f3-cbdbf93a10c5.png)

Reply:
Awesome. I also did a quick run through all pages and all is well.